File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        for(int i=0;i<nums2.size();i++){
            nums1.push_back(nums2[i]);
        }
        sort(nums1.begin(),nums1.end());
        double n=nums1.size();
        double up = nums1[(n/2)+1];
        if(nums1.size()%2==0){
            return (nums1[n/2-1]+nums1[n/2])/2.0;
        }
        else {return nums1[n/2];}
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        ans = sorted(nums1+nums2)
        n = len(ans)
        if n%2!=0:
            return float(ans[n//2])
        else:
            return float(ans[n//2] +ans[n//2-1])/2